lms/static/js/spec/search/search_spec.js

```diff
@@ -611,19 +611,7 @@ define([
                 $('.search-button').trigger('click');
                 // cancel search
                 $('.cancel-button').trigger('click');
-                AjaxHelpers.respondWithJson(requests, {
-                    total: 1337,
-                    access_denied_count: 12,
-                    results: [{
-                        data: {
-                            location: ['section', 'subsection', 'unit'],
-                            url: '/some/url/to/content',
-                            content_type: 'text',
-                            excerpt: 'this is a short excerpt',
-                            course_name: ''
-                        }
-                    }]
-                });
+                AjaxHelpers.skipResetRequest(requests);
```

Comment:
It may be that I need to name the method better (I don't want to have to add this comment for every usage of the method unless it is really necessary). Clicking the cancel button on the line before has reset the request, and then the helper method verifies that the next request was verified and then skips over it. The old code was trying to respond to a reset event which is now a situation that Sinon throws an exception for.